src/test/java/org/broadinstitute/hellbender/tools/PrintReadsIntegrationTest.java

```diff
@@ -31,6 +33,28 @@ public void testFileToFile(String fileIn, String extOut) throws Exception {
         SamAssertionUtils.assertSamsEqual(outFile, ORIG_BAM);
     }
 
+    @Test(dataProvider="testingData")
```

Comment:
We should probably test on CRAM too, but that will fall out once I merge in https://github.com/broadinstitute/gatk/pull/1488 since the CRAM/non-CRAM distinction in these tests will disappear.